go.mod1

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/pingcap/errors v0.11.0
 	github.com/pingcap/tidb v0.0.0-20190218065808-69472bd1a6e9
 	github.com/pingcap/tipb v0.0.0-20190107072121-abbec73437b7
+	github.com/pkg/errors v0.8.1
```

Comment:
cc @tiancaiamao Why is everyone adding this line to `go.mod1`?

Reply:
I find this commit https://github.com/pingcap/parser/commit/60fd134cfde20424ba588c785debb3be0a086210 introduced the usage `github.com/pkg/errors`. Should we replace it with `github.com/pingcap/errors`?